charts/kserve/templates/configmap.yaml

```diff
@@ -38,7 +38,7 @@ data:
       limits:
         cpu: "2"
```

Comment:
Any reason why the default cpu limit for puller and mm sidecar need 2-3 CPU, this adds quite a lot overhead for each modelmesh replica.